File: GARI_Umbria_2026-08-03/GiasNG/src/app/Model/metaschema/Pua.ts
import {BaseCodeDescr} from "../baseClass/baseCodeDescr";
import {Disciplinare} from "./Disciplinari";


export class Pua extends BaseCodeDescr{

    disciplinare: Disciplinare;

    constructor(codice:number, descrizione?: string) {
        super(codice,descrizione);
    }
}

File: GARI_Umbria_2026-08-03/DotNet/AgronicaCoreStd/AgronicaCoreModelsSTD/metaschema/Pua.cs
using AgronicaCoreModelsSTD.baseClass;

namespace AgronicaCoreModelsSTD.metaschema
{
    public class Pua : BaseCodeDescr
    {

        public Disciplinare disciplinare { get; set; }


        public Pua(int codice) : base(codice, "")
        {

        }


        public Pua() : base() { }

    }
}
